querybook/webapp/ui/FlowGraph/QueryCellNode.scss

```diff
@@ -27,6 +27,12 @@
         }
     }
 
```

Comment:
idk when this change was made but can you remove the border change for `&.selected ` (line 13)

and add background change to accent

so instead of border color change, we want a background color change when selected